dev/Dockerfile-dev

```diff
@@ -20,6 +20,7 @@ COPY setup.cfg .
 
 RUN chmod 777 -R .
 RUN chmod 777 /root
+RUN chmod 777 -R /usr/local/lib/
```

Comment:
I added this to be able to install pip packages from inside the docker container. @FinnStutzenstein do know a better way/does this have unwanted side effects?

Reply:
This is fine